corehq/apps/es/tests/utils.py

```diff
@@ -125,23 +125,15 @@ def es_test_decorator(test):
         return es_test_attr(test)
 
     def setup_func():
-        comment = f"created for {test.__module__}.{test.__qualname__}"
-        for adapter in iter_all_doc_adapters():
-            setup_test_index(adapter, comment)
+        for operation in operations.values():
+            operation.run()
 
     def cleanup_func():
-        for adapter in iter_all_doc_adapters():
-            cleanup_test_index(adapter)
+        for operation in operations.values():
+            operation.reverse_run()
 
-    def iter_all_doc_adapters():
-        for adapter in adapters:
-            if isinstance(adapter, ElasticMultiplexAdapter):
-                yield adapter.primary
-                yield adapter.secondary
-            else:
-                yield adapter
-
-    adapters = list(requires)
+    comment = f"created for {test.__module__}.{test.__qualname__}"
```

Comment:
Why use `__qualname__` instead of `__name__`?

I thought `__qualname__` included the module path, so was confused about why this is also adding `test.__module__`, but on testing in a shell I see that `TestCase.__qualname__` simply returns `"TestCase"`, the same as `TestCase.__name__`.

Reply:
I'm not sure I understand. Are you saying you're seeing "wrong" (index) comments, or is the second part of this (PR) comment confirming that you _don't_ see anything "wrong"?

Reply:
I am just curious about why you're using `__qualname__` instead of `__name__` here?

It appears that they both return the same thing, so it should be fine. But I would have chosen `__name__` here since it seems more clear. I think `__qualname__` includes the module path in some contexts (maybe I'm wrong about that?), so it is confusing to see it used in a place that is also prepending the module path.

Reply:
Thanks for clarifying. `__qualname__` does not include the module path, but it does provide [more complete information](https://stackoverflow.com/a/58110139), which can be quite helpful with tests where it's not uncommon to have nested classes and such.

Reply:
Ah, that makes sense. Thanks for clearing up my confusion.